python/py-nameutils/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-nameutils
+version             1.0.0
+revision            0
+
+license             GPL-3+
+platforms           any
+supported_archs     noarch
+maintainers         {raf.org:raf @macportsraf} openmaintainer
+description         Identify given/family names and capitalize correctly
+long_description    {*}${description}. \
+                    \n\nnameutils is a python module containing functions that can split a \
+                    person's full name into their given and family names, and capitalize the \
+                    letters appropriately. It understands complex names in Latin scripts from \
+                    many different languages, and it understands Chinese, Japanese, and Korean \
+                    names, in both their own characters, and romanized.
+
+homepage            https://raf.org/nameutils
+
+checksums           rmd160  0a0fac27a7519b119a616ff6045665bc8fd935b4 \
+                    sha256  1e30bf7420cc74264ecc17a92d12fd2253f9f62ca1949cbbf7c0ffc3e6036bb6 \
+                    size    77846
+
+python.versions     37 38 39 310 311
```

Comment:
if not absolutely needed let's not add EOL Python versions to new ports anymore (i.e., remove PY37)

Reply:
Sure. I just copied what others python ports were doing.